sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -339,9 +336,9 @@ public:
   node add(const property_list& propList = {});
```

Comment:
I don't know if an empty node should require a device or not

Reply:
I think it probably doesn't need one. If we are treating them as barriers depending on the edges to the empty node then it's just up to us during finalize to figure out based on the inputs what command buffers it would affect and handle that accordingly.

There is maybe an argument to be made for having one for API consistency but I don't think that is a strong argument.

Reply:
I second that: device is not required, since no actual command is associated with the empty node. Even if we consider the option to set a CGF for an empty node, it could go along with setting a device.

Reply:
I've rephrased the `syclDevice` parameter to the add() function with a command-group function to be the device `cgf` is processed for.  Therefore, if we don't have a command-group function parameter here for the empty node, I don't think it makes sense to have a device, since there's nothing to use it to process.